### ISSUE TITLE
autocomplete="off" is not sufficient any more

### ISSUE THREAD

**Issue by mokraemer** [jquery/jquery-ui#1982]
e.g. in component autocomplete, the attribute autocomplete is set to off to prevent the browser from autofilling. In many cases the browser ignores this attribute with "off". It looks like setting "one-time-code" will prevent it from autofilling.

**Comment by melloware** [comment]
It loos like that is Apple IOS and Mac only?  https://robindirksen.com/blog/html-autocomplete-one-time-code

**Comment by mokraemer** [comment]
nope. chrome shows autocompletion of predefined user data (e.g. my adress and others). With this setting it disappears.
Most browsers just ignore the setting "off" and handle them as "not set" :(

**Comment by lebbe** [comment]
You need to set autocomplete="off" on the surrounding form tag as well, for Chrome to respect this attribute. Chrome ignores it if only placed on the input element itself (and the developers know about it https://bugs.chromium.org/p/chromium/issues/detail?id=914451).

**Comment by skaestle** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=1133769

**Comment by mokraemer** [comment]
nice post. thanks

**Comment by dominik5566** [comment]
It would be nice if this was fixed. Currently this is the best resource on StackOverflow on this topic
https://stackoverflow.com/questions/15738259/disabling-chrome-autofill

If it is not possible to solve this by setting autocomplete="new-selectbox-item" or similar, then I would suggest to at least not override the attribute if it is already set on the input field.
